packages/rulesets/src/spectral/az-common.ts

```diff
@@ -249,7 +249,7 @@ const ruleset: any = {
       },
     },
     LongRunningOperationsOptionsValidator: {
-      description: 'A LRO Post operation with return schema must have "x-ms-long-running-operation-options" extension enabled.',
+      description: 'A LRO Post operation with return schema should have "x-ms-long-running-operation-options" extension enabled.',
```

Comment:
Minor - I would use `must` for action to take to address an error.

Reply:
This is a warning though, not an error as indicated by severity: "warn",.